style.css

```diff
@@ -14,16 +14,70 @@ body {
   background-color: azure;
 }
 
-.container {
+.nav {
+  height: 60px;
+  background-color: burlywood;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 80px;
+  padding-right: 80px;
+  border: 2px solid bisque;
+  margin: 0 0;
+}
+
+.nav a {
+  text-decoration: none;
+  font-size: 23px;
+  font-weight: 100;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  color: black;
+}
+
+.nav a:hover {
+  color: brown;
+}
+
+.nav-ul {
+  display: flex;
+  list-style: none;
+}
+
+.nav-ul li {
+  margin-left: 40px;
+  font-weight: 100;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  text-decoration: none;
+  color: black;
+  border-left: 2px solid black;
+  padding-left: 20px;
+}
+
+.ul-contact li {
+  font-weight: 100;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+.nav-ul li:hover {
+  color: brown;
+  cursor: pointer;
+}
+
+.all-container {
+  min-height: 100vh;
+}
```

Comment:
- [ ] OPTIONAL
We can obverse that contents of each page are very minimal and therefore, I think we can improve upon the structure/ layout of the application by ensuring that all contents of each page fit the screen without having to scroll to see other contents.

Reply:
Thank you for your feedback. We will work on improving the layout to ensure all content fits on each page without requiring scrolling. 😊